nfd-master.yaml.template

```diff
@@ -0,0 +1,79 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: nfd-master
+  namespace: default
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: nfd-master
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - pods
```

Comment:
Why does the master needs RBAC rules for pods?

Reply:
Hmm, looks like an ancient  leftover/mistake. It shouldn't really be needed. Thanks for spotting this! Fixed in the latest version of the PR